components/brave_shields/browser/brave_shields_web_contents_observer_android.cc

```diff
@@ -19,6 +19,9 @@ namespace brave_shields {
 void BraveShieldsWebContentsObserver::DispatchBlockedEventForWebContents(
     const std::string& block_type, const std::string& subresource,
     WebContents* web_contents) {
+  if (!web_contents) {
+    return;
+  }
```

Comment:
super nit: could you add a newline after this, thanks.

Reply:
BTW, you could skip CI when addressing this comment, thanks.

Reply:
@yrliou thanks, addressed in https://github.com/brave/brave-core/compare/5aa2ac7326caa1855b1841d7c3a045f0e7c2efb3..f6c6202eec8d354bd90c246f9f6d3a6dd0f4a39b